pkgs/by-name/rp/rpcs3/package.nix

```diff
@@ -72,6 +72,8 @@ stdenv.mkDerivation {
     "-DUSE_SYSTEM_PUGIXML=ON"
     "-DUSE_SYSTEM_FLATBUFFERS=ON"
     "-DUSE_SYSTEM_SDL=ON"
+    "-DUSE_SDL=ON"
+    "-DUSE_DISCORD_RPC=ON"
```

Comment:
I would prefer this to be optional. It looks too superflous **and**  it is deprecated.

Reply:
I was not talking about SDL.